driver.py

```diff
@@ -19,7 +19,8 @@ def getConf(storage):
         'port': 443,
         'user': 'UNAME',
         'pass': 'PASS',
-        'last': 'lpost_file'
+        'last': 'lpost_file',
+        'timezone': "3.00"  # Timezone difference to UTC-Z in the form of: "x.yz" or "-x.yz"
```

Comment:
wow such times much zones. I don't think there's much point in non integer ones though. 
that would also get rid of the parser logic in newsreader.

WDYT?

Reply:
It would indeed get rid of the parser logic, but what about our fellow friends living in such `interesting` places? Also, it is a simple comparison and done only once, but I admit it is a bit ugly though. If you think it is really unnecessary, we can assume only integer values.

Reply:
yeah i don't think there's enough value in it, so please do.